hipify_torch/cuda_to_hip_mappings.py

```diff
@@ -587,8 +587,8 @@
             ('include "hip/hip_runtime.h', CONV_INCLUDE_CUDA_MAIN_H, API_DRIVER),
         ),
         (
-            "cuda_runtime.h",
-            ("hip/hip_runtime.h", CONV_INCLUDE_CUDA_MAIN_H, API_RUNTIME),
+            "include <cuda_runtime.h",
```

Comment:
Can you let me know the scenario, where this change is helping?

Reply:
Like said in title, it helps with Transformer Engine hipifying. There is cuda_runtime.h header there that is hipified to hip_runtime.h contrary to <cuda_runtime.h> that translates to <hip/hip_runtime.h>

Reply:
See few occurrences of "cuda_runtime.h" in third_party git submodules, which are currently not built for ROCm. So this changes will work, but I am not happy with this change. 

@jeffdaily @jithunnair-amd ,
Let me know your opinion on this.

Reply:
I just realized that we already follow this approach for `cuda.h` in the lines above, so if anything, this is actually aligning with that approach. I am going to add a change to do this for both the `<>` as well as the `""` variants of include statements.

Reply:
I have committed change for "cuda_runtime.h". Hope it is OK and will help to move on with review

Reply:
Okay, this looks like it should work OK, since the TE header include statements have some prefix such as https://github.com/NVIDIA/TransformerEngine/blob/2d0ab27f6aa2982e35ad371c76aa69ec437c0a49/transformer_engine/jax/csrc/utils.cu#L9